src/build.sh

```diff
@@ -35,15 +35,15 @@ elif [ $RELEASE_BUILD -eq 0 ]; then
 fi
 
 pushd tmx
-cmake -Bbuild -DCMAKE_CXX_FLAGS="${COVERAGE_FLAGS}" -DCMAKE_C_FLAGS="${COVERAGE_FLAGS}" -DCMAKE_BUILD_TYPE="${BUILD_TYPE}" .
+cmake -Bbuild -DCMAKE_PREFIX_PATH=/opt/carma/cmake -DCMAKE_CXX_FLAGS="${COVERAGE_FLAGS}" -DCMAKE_C_FLAGS="${COVERAGE_FLAGS}" -DCMAKE_BUILD_TYPE="${BUILD_TYPE}" .
 pushd build
 make
 make install
 popd
 popd
 
 pushd v2i-hub
-cmake -Bbuild -DqserverPedestrian_DIR=/usr/local/share/qserverPedestrian/cmake -Dv2xhubWebAPI_DIR=/usr/local/share/v2xhubWebAPI/cmake/ -DCMAKE_CXX_FLAGS="${COVERAGE_FLAGS}" -DCMAKE_C_FLAGS="${COVERAGE_FLAGS}" -DCMAKE_BUILD_TYPE="${BUILD_TYPE}" .
+cmake -Bbuild  -DCMAKE_PREFIX_PATH=/opt/carma/cmake -DqserverPedestrian_DIR=/usr/local/share/qserverPedestrian/cmake -Dv2xhubWebAPI_DIR=/usr/local/share/v2xhubWebAPI/cmake/ -DCMAKE_CXX_FLAGS="${COVERAGE_FLAGS}" -DCMAKE_C_FLAGS="${COVERAGE_FLAGS}" -DCMAKE_BUILD_TYPE="${BUILD_TYPE}" .
```

Comment:
What is the purpose of this prefix path.

Reply:
This finds any of the CARMA build libraries (like the time one) that get installed in /opt/carma and lets the CMake find operations find them.